assets/scss/layouts/header/_header.scss

```diff
@@ -132,6 +131,20 @@
     }
 }
 
+.header-logo-position-left {
```

Comment:
Do you think using a modifier might be slightly better? Maybe `header-logo--left`? 🍹